### DISCUSSION TITLE
调整窗口大小，最大化，全屏之后会黑屏

### DISCUSSION THREAD

**Discussion by redtews** [ywmoyue/biliuwp-lite#328] [General]
[20230927.log](https://github.com/ywmoyue/biliuwp-lite/files/12733088/20230927.log)

https://github.com/ywmoyue/biliuwp-lite/assets/146148228/4e4b3a01-0763-4bdc-b75c-70d1ade2d5a9

版本	Windows 10 家庭版
版本号	22H2
安装日期	‎2023/‎2/‎21
操作系统内部版本	19045.3448
体验	Windows Feature Experience Pack 1000.19044.1000.0
系统   : Windows 10 家庭版 64位 版本号 19045.3448 (22H2)
处理器 : Intel Core i7-11800H 核心数 8 线程数 16工艺 10 nm
显卡   : Intel UHD GraphicsNVIDIA GeForce RTX 3060 Laptop DELL 流处理器  256 显存  1G (DDR4 )DELL流处理器  3840 显存  6G (GDDR6 Samsung)
主板   : 0YC2KJ Dell Inc. 芯片组 Intel HM570 
硬盘   : PM991aNVMeSamsung512GB  实际容量477GB 类型SSDSanDiskUltra3DNVMe 实际容量1.82TB类型SSD
显示器 : AU Optronics B156HAN [DELL P/N: 184X2]  分辨率 1920*1080 刷新率165Hz 屏幕尺寸 15.3英寸
内存   : Micron  DDR4-3200   (1600 MHz) 8GBMicron  DDR4-3200   (1600 MHz) 8GB 容量 16 GB通道 2频率 3192 MHz 时序 22-22-22-521T插槽3Micron  DDR4-3200   (1600 MHz) 8GB插槽6Micron  DDR4-3200   (1600 MHz) 8GB
电池   : DELL M59JH15 电量 100 %损耗程度3.4 %

**Comment by ywmoyue** [comment]
@redtews  试试4.6.12或之前的版本是否会出现,或者是否出现的频率没有4.6.12之后那么高

**Reply by redtews** [reply]
G15关闭独显直连后正常了，我还有一台核显的笔记本yoga13s，在上面也没有这个问题，推测可能与独显直连有关，是软件默认调用核显吗？

**Reply by JimHans** [reply]
我这边也是在一次更新以后有一样的问题，但是我这边是台机，只能独显，一直没找到啥好的解决办法。

**Reply by ywmoyue** [reply]
现在正在试着重构播放器的代码，这类播放器相关问题，我后面都研究下

**Comment by ywmoyue** [comment]
@redtews 在4.6.26中已由 @JimHans  修复，麻烦验证一下之后的版本在设置了画面尺寸兼容模式后是否还会出现这个问题

**Reply by ksdh2333** [reply]
使用兼容模式播放，优先独显，依然出现过段时间黑屏只剩弹幕的情况。

**Reply by ywmoyue** [reply]
@ksdh2333  “过段时间黑屏”这个描述看起来和这个讨论不是同一个问题
